dev/io.openliberty.checkpoint.springboot.fat30.webflux.app/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+server.port=8081
```

Comment:
This is missing:
```
logging.level.org.springframework.context.support.DefaultLifecycleProcessor=debug
```

That causes the webflux app to not have the debug out put we expect in the test.

Reply:
done, thanks!